inc/cleantalk-common.php

```diff
@@ -833,6 +836,13 @@ function apbct_is_cache_plugins_exists($return_names = false)
     foreach ($constants_of_cache_plugins as $const => $_text) {
         if ( defined($const) ) {
             $out[] = $_text;
+
+            if ($const === 'SPEEDYCACHE_VERSION') {
```

Comment:
This function is used right for detect cache plugins exists, but not for appliyng any changes in the State
 
I believe we can do this here https://github.com/CleanTalk/wordpress-antispam/blob/94afb61b3b8a47178779055663fd7c00d8421554/lib/Cleantalk/ApbctWP/State.php#L603

Reply:
There will be no information on constants or classes in this place

Reply:
apbct_is_cache_plugins_exists returns names of cache plugins exists if input aprarm is true, maybe compare the name

Reply:
That's right, but no constants or classes will be found in this place yet, and the function will not return anything

Reply:
We decided to remove the switch to alt cookies